### DISCUSSION TITLE
Install stuck

### DISCUSSION THREAD

**Discussion by mamba143** [Security-Onion-Solutions/securityonion#2899] [General]
Hi, I am trying to perform and installation however the install progress is stuck at 0%. it does not matter what option i chose.

Opion selected is eval, distributed and standalone. All methods remain at 0%

**Comment by BustedSec** [comment]
Please provide the output of the /root/sosetup.log

**Reply by bishop340** [reply]
The log is clean for me.

**Reply by bishop340** [reply]
<img width="1026" alt="Screen Shot 2021-03-08 at 10 21 53 AM" src="https://user-images.githubusercontent.com/80270237/110341355-3dc6fc80-7ff8-11eb-87fe-063e030c5f63.png">
<img width="1026" alt="Screen Shot 2021-03-08 at 10 22 34 AM" src="https://user-images.githubusercontent.com/80270237/110341363-40c1ed00-7ff8-11eb-8588-456b5ff72f7b.png">
